CodingStandards/ProgramEntryPoint.cs

```diff
@@ -6,32 +6,42 @@
 using System.Threading.Tasks;
 
 class Program
-{
-        //This is the entry point for the program.
-        static int Main(string[] args)
-{
-    return writeMessage(dtNow, Console.Out);
-}
-
-        static DateTime dtNow = DateTime.Now;
-
-public static int writeMessage(DateTime CurrentDate, TextWriter writer)
-{
-    var intRetVal = 0; const Int32 MESSAGE_COUNT = 5; for (int MyIndex = 0; MyIndex < MESSAGE_COUNT; MyIndex++)
-    /*********************/
-    /* greater than 2016 */
-    /*********************/
-    if (CurrentDate.Year > 2016) writer.WriteLine("Happy New Year!");
-    else
-                writer.WriteLine("Hello From " + CurrentDate.Year);
-
-
-    //--------------------return the result value----------------------------
-
-
-
-
-    return intRetVal;
+{
+    static DateTime DTNow = DateTime.Now;
+
+    // This is the entry point for the program.
+    /// <summary>
+    /// Prints messages to the console
+    /// </summary>
+    /// <param name="args"></param>

```

Comment:
You need to describe the arguments with brief but informative information. Since that wasn't specifically called out in the coding standards, I won't count off this time, but I will next time since XML comments without anything in them is pretty much worthless to other team members.